tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/codegeneration/Metadata.java

```diff
@@ -22,7 +24,7 @@ public class Metadata {
     private String signatureVersion;
     private String signingName;
     private String targetPrefix;
-    private String protocol;
```

Comment:
protocol field is still present on the C2J model files.
moreover, old models, that were not recently updated-or-touched, still have protocol field but not the list.

Reply:
ahhh thought all of them had them